src/js/common/containers/DowntimeNotification.jsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import objectValues from 'lodash/fp/values';
+import { connect } from 'react-redux';
+import { getScheduledDowntime } from '../actions';
+import AlertBox from '../components/AlertBox';
+import Modal from '../components/Modal';
+import LoadingIndicator from '../components/LoadingIndicator';
+
+export const services = {
+  appeals: 'appeals',
+  arcgis: 'arcgis',
+  dslogon: 'dslogon',
+  emis: 'emis',
+  es: 'es',
+  evss: 'evss',
+  idme: 'idme',
+  mvi: 'mvi',
+  mhv: 'mhv',
+  tims: 'tims',
+  vic: 'vic'
+};
+
+export const serviceStatus = {
+  down: 'down',
+  downtimeApproaching: 'downtimeApproaching',
+  ok: 'ok'
+};
+
+function DowntimeNotificationWrapper({ status, children }) {
+  return <div id="downtime-notification" data-status={status} style={{ marginBottom: '1em' }} className="row-padded">{children}</div>;
```

Comment:
No great explanation other than that I really didn't want to add a rule for just that single margin. Think it's worth adding?

Reply:
Yeah, I'm not sure. Is it in a preexisting pattern that may get a style update later? If so, it might be worth it. If not, I wouldn't worry about it.

Reply:
I think it's okay at the moment. If any more style gets added then it'll definitely make sense to move it over but I think it might be staying pretty simple. Hoping to get by using default components/tags.